Client/mods/deathmatch/logic/CModelNames.cpp

```diff
@@ -10017,6 +10017,4421 @@ struct
+    3368,  "lod_bnyd_blk2_",
+    4698,  "LODLAplaza2_LAn",
+    6466,  "pier04Tr_LAw2",
+    8721,  "lodmpic01_lvs",
+    3740,  "LODhillhouse13_LA",
+    18519, "cuntsrod02NEWLOD",
+    10719, "LODundbit_10_SFS",
+    16561, "lod_cen_bit_07",
+    13763, "LODTCEhollyhil10",
+    9987,  "LODA50",
+    16382, "quarry_lodbit01",
+    17377, "lodcuntw21",
+    18593, "lod_cs_landbit_10",
+    9069,  "lodEland36_lvs",
+    18591, "lod_cs_landbit_03",
+    5543,  "LODzroad12",
+    8601,  "lodasEroad019",
+    9467,  "lod_land2_sfN10",
+    17325, "lod_combbarn",
+    8123,  "lodfrates05",
+    4395,  "lodseabed026",
+    17209, "LODcuntwroad52",
+    11307, "LODroadsSFSE41",
+    4978,  "LODedshpblk08",
+    5569,  "LODxroad09",
+    10191, "LODnew_sfe03",
+    18404, "cs_lodbit_41",
+    11036, "LOD_sfs026",
+    9804,  "LODroads_sfw50",
+    17395, "lodcuntw40",
+    4081,  "LODcourt1_LAn",
+    17889, "lod_corner",
+    7235,  "lodNshopchap1",
+    11020, "LOD_sfs006",
+    4929,  "LODcostair_LAS01",
+    3846,  "LOD_hse_13_SFXRF",
+    8029,  "lodasSroad088",
+    9853,  "lod_sfw_boxwest05",
+    8705,  "lodlbr_hotl01_lvS",
+    5468,  "laeLODfers02",
+    9862,  "lod_land_21_sfw",
+    4404,  "lodseabed035",
+    4663,  "LODFreeway4_LAn2",
+    11375, "LODreeway3_SFSe",
+    4096,  "LODsupports06_LAn",
+    17843, "LODLae2_lndHUB02",
+    8021,  "lodasSroad052",
+    11162, "LOD_whouse02_SFS",
+    5206,  "LODpltbild2_las2",
+    7241,  "lodckcafehtl",
+    3736,  "LODbevbrkhus1",
+    8810,  "lodhtclub02_lvs",
+    13248, "LOD03sw_block10",
+    4500,  "lodseabed147",
+    7858,  "lodasNroad24",
+    9979,  "LODA42",
+    9461,  "lod_land2_sfN06",
+    4132,  "LODGSFreeway5_LAn",
+    13202, "LODsw_block05",
+    13770, "LODCE_Roads37",
+    10243, "LODelbits_Sfe05",
+    8764,  "lodEland21_lvs",
+    7823,  "lodagebuild03",
+    5610,  "LODxsJeffers01",
+    5357,  "LODedscrsap_las02",
+    10332, "LODgroundbit_13SFS",
+    11157, "LODetland_SFS",
+    11637, "lod_nw_bit_09",
+    4970,  "LODLAND8_LAS01",
+    7763,  "lodasWedge05",
+    4451,  "lodseabed083",
+    10588, "LODFLATSGND03_SFS",
+    4115,  "LODlanBlocknew2",
+    4580,  "LODkyscrap4_LAn",
+    13832, "LODcunte_roads05",
+    10339, "LODROADSsfs31",
+    10059, "LODA38",
+    7778,  "lodasNroad57",
+    8861,  "lodEcoast05",
+    4956,  "LODrnway5_LAS",
+    6941,  "lodasplant08",
+    11670, "lod_des_adobech",
+    6870,  "lodckcafe",
+    9942,  "LODA09",
+    13291, "LODcuntetownrd1",
+    7728,  "lodballparkland",
+    10540, "LODROADSsfs37",
+    9755,  "LODp_SFW35",
+    10690, "LODtrack02_SFS",
+    17391, "lodcuntw36",
+    4050,  "LODfigfree1_LAn",
+    17966, "LODgangshps1d",
+    9161,  "lodasEroad138",
+    10279, "LODstu69c_sfe",
+    6241,  "LODcanal_floor2",
+    10518, "LOD_sfs055",
+    18511, "LODcunts12",
+    10331, "LODgroundbit_12SFS",
+    3725,  "LODhillhouse06_LA",
+    7255,  "lodasNedge22",
+    8892,  "lodbond03_lvs",
+//  16562, "cn2_rosmot1",
```

Comment:
Could you add some info to the ones you commented out eg. "Wrong name retained for BC"